src/rabbit_queue_master_location_misc.erl

```diff
@@ -0,0 +1,144 @@
+%%% The contents of this file are subject to the Mozilla Public License
+%%% Version 1.1 (the "License"); you may not use this file except in
+%%% compliance with the License. You may obtain a copy of the License at
+%%% http://www.mozilla.org/MPL/
+%%%
+%%% Software distributed under the License is distributed on an "AS IS"
+%%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+%%% License for the specific language governing rights and limitations
+%%% under the License.
+%%%
+%%% The Original Code is RabbitMQ.
+%%%
+%%% @author Ayanda Dube <ayanda.dube@erlang-solutions.com>
+%%% @doc
+%%% - Queue Master Location miscellaneous functions
+%%%
+%%% @end
+%%% Created : 19. Jun 2015
+%%%-------------------------------------------------------------------
+-module(rabbit_queue_master_location_misc).
+-behaviour(rabbit_policy_validator).
+
+-include("rabbit.hrl").
+
+%% API
+-export([ lookup_master/2,
+          lookup_queue/2,
+          get_location/1,
+          get_location_by_config/1,
+          get_location_by_args/1,
+          get_location_by_policy/1,
+          all_nodes/0,
+          delay_ms/1,
```

Comment:
this is not used, besides it's the same as http://www.erlang.org/doc/man/timer.html#sleep-1